Applications/MIDIMonitor/SMMAppController.h

```diff
@@ -19,4 +19,7 @@
 @end
 
 // Preference keys
-extern NSString* const SMMOpenWindowsForNewSourcesPreferenceKey;
+extern NSString* const SMMAutoConnectNewSourcesPreferenceKey;
+//extern NSString* const SMMOpenWindowsForNewSourcesPreferenceKey;
```

Comment:
Please delete this, there is no need to leave commented-out code around.

Reply:
(Although you probably want to keep the old preference key around for migration purposes.)

Reply:
Would you prefer I just delete the one comment but leave the old OpenWindowsForNewSourcesPreferenceKey?

Reply:
Right, leave the old one in place, not commented out, since you'll need it for migration.